ops/terraform/modules/resources/iam/main.tf

```diff
@@ -102,8 +102,37 @@ resource "aws_iam_policy" "ssm" {
 EOF
 }
 
-# attach AWS managed CloudWatchAgentServerPolicy to all EC2 instances
+# attach AWS managed SSM paramaters to all EC2 instances
```

Comment:
👍 Thanks for catching the copy/pasta error. Let's solve for the typo while we're at it.
```suggestion
# attach AWS managed SSM parameters to all EC2 instances
```

Reply:
Will fix (mainly by making sure my spell check is turned on for all file type) Thanks for the keen eye. 😃